index.js

```diff
@@ -0,0 +1,230 @@
+      date0fExpiry: "11/11/2022",
+      sugarFree: true,
+    },
+    productProvider: "FOP Piggy",
+    productCountry: "https://www.worldometers.info/img/flags/up-flag.gif",
+    productPrice: 232,
+  },
+  {
+    productName: "wattermelon",
+    productData: {
+      weight: 200,
+      certificate: true,
+      date0fExpiry: "20/10/2022",
+      sugarFree: true,
+    },
+    productProvider: "TOV Herson-Ukraine",
+    productCountry: "https://www.worldometers.info/img/flags/up-flag.gif",
+    productPrice: 77,
+  },
+  {
+    productName: "Salt",
+    productData: {
+      weight: 200,
+      certificate: true,
+      date0fExpiry: "21/10/2023",
+      sugarFree: true,
+    },
+    productProvider: "FOP Artem",
+    productCountry: "https://www.worldometers.info/img/flags/up-flag.gif",
+    productPrice: 7,
+  },
+  {
+    productName: "potato",
+    productData: {
+      weight: 200,
+      certificate: true,
+      date0fExpiry: "04/11/2022",
+      sugarFree: true,
+    },
+    productProvider: "FOP Potato",
+    productCountry: "https://www.worldometers.info/img/flags/up-flag.gif",
+    productPrice: 12,
+  },
+  {
+    productName: "Lamb",
+    productData: {
+      weight: 100,
+      certificate: true,
+      date0fExpiry: "26/10/2022",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "https://www.worldometers.info/img/flags/up-flag.gif",
+    productPrice: 300,
+  },
+  {
+    productName: "Onion",
+    productData: {
+      weight: 400,
+      certificate: true,
+      date0fExpiry: "21/12/2022",
+      sugarFree: true,
+    },
+    productProvider: "FOP ON I ON",
+    productCountry: "https://www.worldometers.info/img/flags/up-flag.gif",
+    productPrice: 3,
+  },
+  {
+    productName: "Carrot",
+    productData: {
+      weight: 220,
+      certificate: true,
+      date0fExpiry: "29/11/2022",
+      sugarFree: true,
+    },
+    productProvider: "FOP CAR",
+    productCountry: "https://www.worldometers.info/img/flags/up-flag.gif",
+    productPrice: 21,
+  },
+  {
+    productName: "Paprika",
+    productData: {
+      weight: 200,
+      certificate: true,
+      date0fExpiry: "02/10/2022",
+      sugarFree: true,
+    },
+    productProvider: "FOP PAP",
+    productCountry: "https://www.worldometers.info/img/flags/up-flag.gif",
+    productPrice: 19,
+  },
+  {
+    productName: "Juice",
+    productData: {
+      weight: 100,
+      certificate: true,
+      date0fExpiry: "11/12/2023",
+      sugarFree: true,
+    },
+    productProvider: "Tov SANDORA",
+    productCountry: "https://www.worldometers.info/img/flags/up-flag.gif",
+    productPrice: 50,
+  },
+];
+console.log(productsList);
+
+const productsRoll = document.createElement("ul");
+function createProduct(productsList) {
+  const itemsList = document.createElement("li");
+  const productInfo = document.createElement("p");
+  const productIMG = document.createElement("img");
+  itemsList.classList.add("list");
```

Comment:
```suggestion
  itemsList.classList.add("listItem");
```
Looks more meaningful